dev-manuals/architecture/authorization-concept.md

```diff
@@ -9,6 +9,18 @@ This document outlines the authorization concept used to ensure that graphql que
 * There are different permission levels ("roles", not to be confused with the concept in Keycloak also called roles) both on the global and on the course scale. E.g. a lecturer of a course has write-permissions in that course, but not in other courses; A student assigned to a course has read permissions only for that course. Permissions cannot be set fine-grained per user, only a role with a predefined set of permissions (either globally or on a course-specific level) can be assigned to a user
 * We assume there are no additional hierarchical permission concepts other than global permissions and course-specific permissions. This means that it is not possible for example for a user to only have permissions for some contents of a course. You either have permissions for everything that's part of the course, or you do not. (Of course this limitation does not impede on the ability of services to implement specific limitations on access to course content which is not dependent on independent user permissions but instead on other rules, e.g. a quiz which can only be accessed by students before a specific date has passed)
 
+## User Roles
+### Global Roles
+Administrator: Has complete control over the application, including user management, course creation, and overall system configuration.
+Instructor: Can create and manage courses, upload course materials, create assessments, and grade student submissions.
```

Comment:
Did we not say, that we don't want the global roles to sound like study related roles (here "Instructor" and "Learner")?

Reply:
Don't remember clearly, thought it was "Not university or school" related. For the off chance, it is used as a learning platform outside the the university/school space.